specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2025-05-01/ResourceHealth.json

```diff
@@ -1360,6 +1418,25 @@
             "impactType": {
               "type": "string",
               "description": "The type of the impact"
+            },
+            "eventTags": {
+              "$ref": "#/definitions/eventTags"
+            },
+            "newRate": {
+              "type": "string",
+              "description": "Billing rate change information - new rate"
```

Comment:
I'm curious about why they are strings rather than numbers

Reply:
There isn't a hard requirement for these to be strings. We do not support any arithmetic operations on these fields, and this data is for informational purposes it does not represent the actual billing amount.

Reply:
I am not sure I understand what you're saying, about arithmetic and billing amounts.

It sounds like maybe you are stating that you have some assumptions, that the API is going to be consumed in limited ways?

But the way you said it, I am tempted to think that you worry if the numbers you are returning are even accurate and trustworthy and can be used by real people for calculations, even to estimate their bill? 
I hope that is not a genuine concern?! Or at least if that were the case, there'd be appropriate disclaimers somewhere?

I think when we're doing API design we don't want to make too many assumptions about how the API would be used in the future. (For instance, its only getting easier and easier to use AI to consume api docs and write new client apps.)

Reply:
Yes these numbers are not for customers to calculate the cost incurred for their azure usage. We can make it a double

Reply:
Changed the type to number / double.